haskell/test/Test/Ligo/BaseDAO/Common.hs

```diff
@@ -122,32 +123,32 @@ permitProtect author (toSign, a) = do
 
 sendXtz
   :: (MonadCleveland caps m, HasCallStack)
-  => TAddress cp vd -> m ()
+  => ContractHandle cp st vd  -> m ()
 sendXtz addr = withFrozenCallStack $ do
   let transferData = TransferData
-        { tdTo = toAddress addr
+        { tdTo = addr
         , tdAmount = [tz|0.5|] -- 0.5 xtz
         , tdEntrypoint = DefEpName
         , tdParameter = ()
         }
-  transfer transferData
+  void $ runTransfer transferData
```

Comment:
Why not just

```haskell
transfer addr [tz|0.5|]
```
?

Reply:
Seems to throw this error...

```
• Couldn't match type ‘first-class-families-0.8.0.1:Fcf.Core.Eval
||                              (first-class-families-0.8.0.1:Fcf.Data.Common.FromMaybe
||                                 cp
||                                 (first-class-families-0.8.0.1:Fcf.Core.Eval
||                                    (EpdLookupEntrypoint
||                                       (Data.Type.Bool.If
||                                          (Lorentz.Entrypoints.Helpers.CanHaveEntrypoints cp)
||                                          (ParameterEntrypointsDerivation cp)
||                                          EpdNone)
||                                       cp
||                                       Lorentz.Entrypoints.Core.DefaultEpName)))’
||                      with ‘()’
||         arising from a use of ‘transfer’
```

Reply:
Ah, indeed, that would be the case here, it needs `HasDefEntrypointArg cp ep ()` constraint. The error message is very arcane though, I need to remember to file an issue on morley.

This happens because `ContractHandle` is checked by default. The alternative is to throw away typing information by using `transfer (chAddress addr) [tz|0.5|]` for instance.

Reply:
Ok, I ll leave it as it is for now.

Reply:
Just use `transfer (chAddress addr) [tz|0.5|]`.